db/fixtures/ae_datastore/ManageIQ/Cloud/VM/Retirement/StateMachines/Methods.class/__methods__/microsoft_pre_retirement.rb

```diff
@@ -0,0 +1,10 @@
+#
```

Comment:
@tinaafitz @bzwei The idea was to have provider specific methods so users could easily override the logic by just overriding this method.  Checking to see if a VM is stopped is pretty generic because we reference a normalized value and is unlikely to change so that can be common logic.

However, it seems more likely that a user will want to customize how you stop (suspend, stop, terminate, shelve) the VM and be different per provider.

Thoughts?